components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher/src/gen/java/org/wso2/carbon/apimgt/rest/api/publisher/CertificatesApi.java

```diff
@@ -0,0 +1,149 @@
+package org.wso2.carbon.apimgt.rest.api.publisher;
+
+import org.wso2.carbon.apimgt.rest.api.publisher.dto.*;
```

Comment:
I think, its better you can import the specific package rather importing the whole.

Reply:
Please ignore the comment since the codes are generated by swagger maven plugin.